pkgs/data/fonts/apple-color-emoji/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "apple-color-emoji";
+
+  src = fetchurl {
+    url = https://github.com/samuelngs/apple-emoji-linux/releases/download/latest/AppleColorEmoji.ttf;
+    sha256 = "0xaclj29b7xgqin8izkabrm2znp1m01894fngyxrhwbf9nbncp4g";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  sourceRoot = "./";
+
+  unpackCmd = ''
+    cp $curSrc ./AppleColorEmoji.ttf
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/apple-color-emoji
+    cp *.ttf $out/share/fonts/apple-color-emoji
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A Color typeface used by iOS and macOS to display emoji";
+    homepage = https://github.com/samuelngs/apple-emoji-linux;
```

Please quote URLs. See https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md
```suggestion
    homepage = "https://github.com/samuelngs/apple-emoji-linux";
```